docs/pages/setup/reference/cli.mdx

```diff
@@ -685,6 +685,54 @@ registered devices using [`tsh mfa ls`](cli.mdx#tsh-mfa-ls).
 $ tsh mfa rm <device-name>
 ```
 
+### tsh exec
+
+Run provided command, by default using a shell, with Teleport-specific environment
```

Comment:
I think the first paragraph needs to make a more general statement about why `tsh exec` exists and when it is used. The rest of this reference entry mentions the use of a shell and the Teleport-specific environment variables, so I don't think we need to mention them here. 

If using the provided shell and adding the Teleport-specific environment variables _are_ the purposes of the `tsh exec` command, though, I think we can clarify that a bit more.

To start, what's the difference between using:

```bash
$ tsh exec --shell bash -- -c 'echo "${TSH} ${TSH_COMMAND}"'
```
And:

```BASH
$ export TSH=foo; export TSH_COMMAND=bar; bash -c 'echo "${TSH} ${TSH_COMMAND}"'

Reply:
A large part of motivation for `tsh exec` are tsh aliases, which are not done yet. 

Outside of that, there is nothing **strictly** new that wasn't possible before: you can write your own scripts, run `tsh env` to get the variables, run `where tsh` to find path to `tsh` binary etc. Yet `tsh exec` automates all of that, which is a convenience.